packages/experimental/Divider/src/Divider.styling.ts

```diff
@@ -0,0 +1,104 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { layoutStyles } from '@fluentui-react-native/tokens';
+import { DividerTokens, DividerSlotProps, DividerAppearance } from './Divider.types';
+import { IconPropsV1 } from '@fluentui-react-native/icon';
```

Comment:
import { IconPropsV1 as IconProps } from '@fluentui-react-native/icon';

Reply:
This way, you only have to update this import when we eventually rename IconPropsV1 to IconProps